net-misc/streamlink/streamlink-0.1.0.ebuild

```diff
@@ -1,39 +0,0 @@
-# Copyright 1999-2016 Gentoo Foundation
```

Comment:
First you remove the last keyworded version from tree leaving a dangling Manifest entry behind. Then proceed with the bump to reintroduce a keyworded ebuild and update Manifest.

Even though this will be pushed in one go, it would be nice to avoid tree breakage at any given point and first do bump and only then remove previous version. Coud you please reorder your commits and adjust Manifest handling.